cmd/controller/main.go

```diff
@@ -98,10 +98,19 @@ func myNamespace() string {
 	return metav1.NamespaceDefault
 }
 
-// Initialises the first key and starts the rotation job. returns an early trigger function
+// Initialises the first key and starts the rotation job. returns an early trigger function.
+// A period of 0 disables automatic rotation, but manual rotation (e.g. triggered by signal) is still
```

Comment:
Can you clarify what you mean by "signal" ?

Reply:
PTAL